content/java/concepts/math-methods/terms/atan/atan.md

```diff
@@ -11,11 +11,11 @@ CatalogContent:
   - 'paths/computer-science'
 ---
 
-Returns the inverse tangent of the argument in radians.
+The **`Math.atan()`** method returns the inverse tangent of the argument in radians.
 
 ## Syntax
 
-```java
+```pseudo
 Math.atan(n)
 ```
 
```

Comment:
Can we add a brief sentence here describing what `n` is (including what data type(s))?

Reply:
Also, on line # 24, can we rephrase the sentence to something like this?

```
The following example uses `Math.atan()` to return the inverse tangent of `1.0`:
```